lib/src/main/java/adminlte/entity_list_table/communication/http/tables/AbstractTable.java

```diff
@@ -46,4 +44,55 @@ protected void addColumn(ColumnDefinitionInterface column) {
     public String getPaginationParameter() { return this.paginationParameter; }
 
     public String getSearchParameter() { return this.searchParameter; }
+
+    public List<ColumnDefinitionInterface> getCsvColumns() { return new ArrayList<ColumnDefinitionInterface>(); }
+
+    public String generateCsv(String fileName, boolean addQuotation) {
```

Comment:
This is bad. Formatting CSV is not part of the AbstractTable class responsibility. I think that we should work with the service's renderTable and TableRender class and provide another layer of abstraction (like TableHtmlRenderer / TableCsvRenderer), or maybe even create a new service to handle CSV exports.

Reply:
Expanded upon this in next commit